components/ExerciseCard/ExerciseCard.tsx

```diff
@@ -48,6 +58,11 @@ const ExerciseCard = ({
               message === Message.ERROR ? styles.exerciseCard__input__error : ''
             }`}
             value={studentAnswer}
+            onKeyDown={e => {
+              if (e.key === 'Enter') {
+                submitAnswer()
+              }
+            }}
```

Comment:
Another possible way to do this would be to wrap the exercise card in a form and then selectively choose which buttons cause the form to submit. For example, the "Show/hide answer" button shouldn't cause the form to submit.